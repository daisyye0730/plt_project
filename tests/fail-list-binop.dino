int main() {
    List(int, 3) li1;
    List(int, 3) li2;
    List(int, 4) sum;
    int i;

    li1 = [2, 3, 4];
    li2 = [-2, -3, -4];

    sum = li1+li2;

    for (i = 0; i < 3; i++) {
      print(sum[i]);
    }
}