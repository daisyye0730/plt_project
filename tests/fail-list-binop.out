Fatal error: exception Failure("illegal assignment List(int, 4) = List(int, 3) in sum = li1 + li2")
